image_provisioner/playbooks/roles/atomic-kickstart/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+---
+- name: download ssh keys
+  get_url:
+    url: https://api.github.com/repos/redhat-performance{{ ssh_repo }}
+    dest: /root/.ssh/id_rsa
+    mode: 0600
+    headers: 'Authorization:token {{ github_token }},Accept:application/vnd.github.v3.raw'
+
+- name: create directory to store pbench results
+  file: 
+    path: /var/lib/pbench-agent 
+    state: directory
+
+- name: create directory to store openshift templates for pbench
+  file: 
+    path: /var/home/openshift-templates/pbench 
+    state: directory
+
+- name: copy openshift template for pbench
+  copy: 
+    src: /root/svt/image_provisioner/openshift-templates/pbench/pbench-agent-daemonset.yml  
+    dest: /var/home/openshift-templates/pbench 
+    owner: root 
+    group: root 
+    mode: 0644
+
+- name: build pbench-agent image
+  docker_image:
+    path: /root/svt/image_provisioner/dockerfiles/pbench-agent
+    name: pbench-agent
+  delegate_to: localhost
+
+- name: archive pbench-agent image
+  docker_image:
+    name: pbench-agent
+    archive_path: /var/tmp/pbench-agent.tar
+  delegate_to: localhost
+
+- name: copy pbench-agent image to remote atomic host
+  copy: 
+    src: /var/tmp/pbench-agent.tar 
+    dest: /var/tmp/pbench-agent.tar 
+    owner: root 
+    group: root 
+    mode: 0644
+
+- name: delete the pbench-agent archive on the local machine
+  file: 
+    path: /var/tmp/pbench-agent.tar 
+    state: absent
+  delegate_to: localhost
+
+- name: build pbench-controller image
+  docker_image:
+    path: /root/svt/image_provisioner/dockerfiles/pbench-controller
+    name: pbench-controller
+  delegate_to: localhost
+
+- name: archive pbench-controller image
+  shell: docker save pbench-controller > /var/tmp/controller.tar 
+  delegate_to: localhost
+
+- name: copy pbench-controller image to remote atomic host
+  copy: 
+    src: /var/tmp/controller.tar 
+    dest: /var/tmp/controller.tar 
+    owner: root 
+    group: root 
+    mode: 0644
+
+- name: delete the pbench-controller archive on the local machine
+  file: 
+    path: /var/tmp/controller.tar 
+    state: absent
+  delegate_to: localhost
+
+- name: load pbench-agent image from archive
+  docker_image:
+    name: pbench-controller
+    load_path: /var/tmp/pbench-agent.tar
+
+- name: load pbench-controller from archive
+  shell: docker load -i /var/tmp/controller.tar
```

Comment:
docker_image module times out if the image size is large, setting COMPOSE_HTTP_TIMEOUT=50000 doesn't work either.